core-web/libs/ui/src/lib/components/dot-workflow-actions/dot-workflow-actions.component.html

```diff
@@ -1,18 +1,20 @@
-@for (action of groupedActions(); track $index) {
-    @if (action.length > 1) {
+@for (action of $groupedActions(); track $index) {
+    @let mainAction = action.mainAction;
+    @let subActions = action.subActions;
+    @if (subActions.length) {
         <p-splitButton
-            (onClick)="action[0].command({ originalEvent: $event })"
+            (onClick)="mainAction.command({ originalEvent: $event })"
```

Comment:
According to strict checking the command function could be undefined, I suggest using nullish coalescing like this:

`mainAction?.command({ originalEvent: $event })`

Reply:
Done!